compiler-rt/lib/msan/msan.cpp

```diff
@@ -352,10 +352,46 @@ void __sanitizer::BufferedStackTrace::UnwindImpl(
 
 using namespace __msan;
 
+static inline void PrintFaultingInstruction(char *instname) {
```

Comment:
what about the other part of my comment? i think it is a bit confusing that a function that says `PrintSomething` is actually conditional

Reply:
I've renamed PrintFaultingInstruction to PrintFaultingInstructionIfRequested.